src/Agda/Convert.hs

```diff
@@ -302,7 +312,12 @@ lispifyGoalSpecificDisplayInfo ii kind = localTCState $
 
         auxSect <- case aux of
           GoalOnly -> return []
+#if MIN_VERSION_Agda(2,6,4)
+          GoalAndHave expr bndry -> do
+            -- TODO: render bndry
+#else
```

Comment:
@banacorn:  Here I do not really know how to render the boundary.  It would be good to be able to test the output before releasing some rendering.  How would one test this?

Reply:
Let me see if I can release a new version of agda-mode on the VS Code Marketplace so that you can test this with ease!

Reply:
I've just released [agda-mode v.0.4.4](https://marketplace.visualstudio.com/items?itemName=banacorn.agda-mode)! 

To test the language server:
1. Build and install it on your machine so that `als` is in PATH
2. [Enable `agdaMode.connection.agdaLanguageServer` in the settings](https://github.com/banacorn/agda-mode-vscode#agda-language-server)
3. If all goes well, you should see "ALS-2.6.4" on the top right corner of the panel. <img width="290" alt="Screenshot 2023-12-03 at 16 47 26" src="https://github.com/agda/agda-language-server/assets/797844/b3c50d2c-7656-471b-b712-fb812a478439">

You should also see errors like `LSP: no handler for: STextDocumentDidOpen` popping up.

Reply:
I've found the reason why there's [this TCP tunneling thing](https://github.com/agda/agda-language-server/blob/master/src/Server.hs#L40C1-L54) on the server. It allow us to develop the language server right in the REPL without having to recompile and reinstall it.

See https://github.com/agda/agda-language-server#hacking for more.